framework/source/class/qx/ui/tree/VirtualTree.js

```diff
@@ -608,7 +608,7 @@ qx.Class.define("qx.ui.tree.VirtualTree",
     
     // Interface implementation
     isShowTopLevelOpenCloseIcons : function() {
-      return true;
+      return this.getShowTopLevelOpenCloseIcons();
```

Comment:
I believe this whole function is unnecessary. I believe it is overriding the function of the same name which is automatically-generated for boolean properties. I believe we should probably remove this method entirely, if my recollection of the automatically-created methods is accurate...

Reply:
Yes, I agree. I've removed the whole method. Thanks!